zip-zcash-grants-fund.rst

```diff
@@ -0,0 +1,330 @@
+   to operate the Community Advisory Panel and SHOULD work toward making it more
+   representative and independent (more on that below).
+   
+   Zcash Community Grants Committee members are expected to work approximately 35
+   hours per month and will be compensated accordingly from the Zcash Community Grants Committee
+   budget. The total compensation for the committee is paid from the Zcash Community Grants Committee budget. 
+   This works out to be a total of 175hours of compensation per month which is roughly the equivalent of a 
+   signle full time position.
+   
+8. A portion of the funds shall be allocated to a
+   Discretionary Budget, which may be disbursed for expenses reasonably related
+   to the administration of Zcash Community Grants. The amount of funds allocated to the
+   Discretionary Budget SHALL be decided by the ZF's Community Advisory Panel or
+   successor process. Any disbursement of funds from the Discretionary Budget
+   MUST be approved by the Zcash Community Grants Committee. Expenses related to the
+   administration of Zcash Community Grants include, without limitation the following:
+   
+   * Paying third party vendors for services related to domain name registration, or
+     the design, website hosting and administration of websites for the Zcash Community Grants
+     Committee.
+   * Paying independent consultants to develop requests for proposals that align
+     with the Zcash Community Grants program.
+   * Paying independent consultants for expert review of grant applications.
+   * Paying for sales and marketing services to promote the Zcash Community Grants
+     program.
+   * Paying third party consultants to undertake activities that support the
+     purpose of the Zcash Community Grants program. 
+   * Reimbursement to members of the Zcash Community Grants Committee for reasonable
+     travel expenses, including transportation, hotel and meals allowance.
+
+9. A portion of the Discretionary Budget MAY be allocated to provide reasonable
+   compensation to members of the ZCG Committee. Committee member compensation
+   SHALL be limited to the hours needed to successfully perform their positions
+   and MUST align with the scope and responsibilities of their roles. The
+   allocation and distribution of compensation to committee members SHALL be
+   administered by FPF. The compensation rate and hours for committee members SHALL be determined by
+   the ZF’s Community Advisory Panel or successor process.
+
+FPF SHALL recognize the ZCG allocation of the Grants Fund as a Restricted Fund
+donation under the above constraints (suitably formalized), and keep separate
+accounting of its balance and usage under its `Transparency and Accountability`_
+obligations defined below.
+
+
+Transparency and Accountability
+-------------------------------
+
+Obligations
+~~~~~~~~~~~
+
+ZCG (during and leading to their award
+period) SHALL all accept the obligations in this section.

```

Comment:
"award period" is not defined, is this supposed to be the period of the funding stream? If so please make it clearer